Model/inverted-index.js

```diff
@@ -2,24 +2,17 @@ class Index {
   constructor() {
     this.index = {};
     this.temp_search = [];
+    this.searchResult = {};
+    this.allIndex = {};
   }
-  // isValid(arrObject) {
-  //   if (typeof arrObject === 'object') {
-  //     const len = Object.keys(arrObject).length;
-  //     if (len > 0) {
-  //       for (const key in arrObject) {
-  //         if (!arrObject[key].hasOwnProperty('title') && !arrObject[key].hasOwnProperty('text')) {
-  //           throw new Error('Dosent have title and text');
-  //         }
-  //       }
-  //     }
-  //   }
-  //   return true;
-  // }
 
   tokenize(words) {
-    const word = words.toLowerCase();
-    return word.trim().replace(/[^a-zA-Z 0-9]+/g, '');
+    const token = words.replace(/,+/g, ' ')
+    .replace(/[^a-zA-Z 0-9\s]+/g, '')
+    .replace(/\s\s/g, ' ')
```

Comment:
Great work, but I'm curious why are you using three different regex replace() here?